### PR TITLE
Add drone pipeline to check the oc10 daily tarball build

### PR DESCRIPTION
**Pull request by saw-jan**
Added CI pipeline to check whether the daily-master-qa tarball is up to date or not. The tarball is considered up to date if its build date is `today` or `yesterday` 's date.

Test runs:
Passing: https://drone.owncloud.com/owncloud/activity/3118/1/2
Failing: https://drone.owncloud.com/owncloud/activity/3117/1/2

Fixes https://github.com/owncloud/QA/issues/760

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -2340,3 +2340,27 @@ def skipIfUnchanged(ctx, type):
...
+            "name": "check-daily-tarball",
+            "image": OC_CI_ALPINE,
+            "commands": [
+                "chmod +x ./tests/drone/validate-oc-daily-tarball.sh",
+                "./tests/drone/validate-oc-daily-tarball.sh",
+            ],
+        }],
+        "depends_on": [],
+        "trigger": {
+            "ref": [],
```
Comment (saw-jan) [inline .drone.star:2359]: Do we want to check the tarball build date in PRs and tags?

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUICreateUpdat-master-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/activity/3115/15

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIDelRestore-master-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/activity/3115/17

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUITags-master-firefox-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/activity/3115/22

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUICreateUpdat-master-firefox-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/activity/3115/16

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIMoveRename-master-firefox-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/activity/3115/24

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiActivity-master-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/activity/3115/27

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUISharingInt-master-firefox-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/activity/3115/20

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIComments-master-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/activity/3115/13

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUISharingInt-master-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/activity/3115/19

File: .drone.star [hunk 2]
```diff
@@ -2340,3 +2340,31 @@ def skipIfUnchanged(ctx, type):
...
     return []
+
+def validateDailyTarballBuild():
+    pipeline = {
+        "kind": "pipeline",
+        "type": "docker",
+        "name": "validate-daily-tarball-build",
+        "environment": {
+            # provide multiple values separated by comma
+            "DAILY_TARBALLS": "daily-master,daily-master-qa",
```
Comment (saw-jan) [inline .drone.star:2351]: Now I think it would be better to pass tarballs as args to the bash file.

**Pull request merged**